test/e2e/content/case.nothing-to-change.test.js

```diff
@@ -33,4 +36,9 @@ describe("When the page does not include any gender pronouns or stopwords, it",
         const headerText = await page.$eval(headerSelector, e => e.innerText);
         expect(headerText).to.include("no gender pronouns");
     });
+
+    it("should make replacements even when hidden elements contain the word 'gender'", async function() {
+        const replacements = visibilityPage.$$(replacementSelector);
+        expect(replacements).to.be.empty;
+    });
 });
```

Comment:
This is confusing. The description is "it should make replacements", but the test is "expect(replacements).to.be.empty". Please clarify.

Reply:
Is a second test needed where pronouns are actually replaced? I'm fairly convinced this one test is adequate to ensure the right code path is being taken, but you should be convinced as well.

If you do think a second test is needed, that one would go in `test/e2e/content/case.with-gender-pronouns.test.js`

Reply:
If a single test, it is probably more appropriate to put it in a new file named something like `case.mentions-gender-invisibly.js`. The new test you wrote has no code in common with the existing tests in `case.nothing-to-change.test.js`, and I would argue it is actually a new use case. I don't mind proliferating files if it keeps the intention behind each file clearer.

Reply:
Or, perhaps better, all of the tests in his case could use the new page. The tests that were originally there would be equally applicable to the new page you created. You might rename it accordingly.